lib/puppet/application/cert.rb

```diff
@@ -265,8 +265,10 @@ def main
     end
     begin
       if subcommand == :destroy
+        raise _("Refusing to destroy all certs, provide an explicit list of certs to destroy") if hosts == :all
```

Comment:
I didn't think we had i18n support in 4.10.x?

Reply:
@ericldelaney might also have a solid answer. I just remember all his prs were targeted at 5.3.x

Reply:
I've been confused about this too... We actually did introduce i18n support in the 4.x series as there are `_(..)` strings and https://github.com/puppetlabs/puppet/blob/4.10.x/locales/puppet.pot. There's also a localization job `platform_puppet_puppet-agent-component-localization_4.10.x_agent-1.10.x/.` I think the confusion is that the i18n process began in 4.x, but spanned multiple releases (and still continues).

Reply:
I believe we have the machinery or stubs in 4.10.x but we're not doing translations there. You can add decorated strings, but nothing happens with them, Its only in 5.3.x and up that we actually will translate strings

Reply:
Thanks for the context @ericldelaney. Since this code will be merged to 5.3.x and up, I think it makes sense to internationalize it here.